emukit/sensitivity/monte_carlo/monte_carlo_sensitivity.py

```diff
@@ -99,3 +99,22 @@ def compute_effects(self, main_sample: np.ndarray=None, fixing_sample: np.ndarra
 
             var_index += 1
         return main_effects, total_effects, total_variance
+
+
+
+class ModelBasedMonteCarloSensitivity(MonteCarloSensitivity):
+    '''
+    Class to compute the sensitivity coefficients of a Gaussian process  This class wraps the model and calls the mean
```

Comment:
This doesn't seem to be GP specific but the comment makes it seem that way.

Reply:
You are totally right, this class is general, I'll change that.